magefile.go

```diff
@@ -0,0 +1,299 @@
+// Copyright 2023 uwu-tools Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build mage
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/magefile/mage/sh"
+	"sigs.k8s.io/release-utils/mage"
+
+	"github.com/uwu-tools/magex/pkg"
+)
+
+// Default target to run when none is specified
+// If not set, running mage will list available targets
+var Default = Verify
+
+const (
+	binDir     = "bin"
+	moduleName = "github.com/uwu-tools/gh-jira-issue-sync"
+	scriptDir  = "scripts"
+
+	// Versions.
+	golangciVersion = "v1.50.1"
+	minGoVersion    = "1.20"
+
+	// Environment variables.
+	envLDFLAGS      = "GHJIRA_LDFLAGS"
+	envMinGoVersion = "MIN_GO_VERSION"
+
+	// Test variables.
+	coverMode            = "atomic"
+	coverProfileFilename = "unit-coverage.out"
+)
+
+// All runs all targets for this repository
```

Comment:
The description is a little bit misleading, since we only run two targets not all of them